requirements.txt

```diff
@@ -1,3 +1,10 @@
-numpy
 h5py
+nose
+numpy
+scipy
+setuptools
+six
+typing
 typing
+wheel
+
```

Comment:
Don't include this in this pull request.
Also, in general, this should not be needed. Esp it should not contain stuff which are only needed for the tests.
Numpy should also be covered already when you install TF (TF is not in there because some people would use `tensorflow-gpu`, while others use just `tensorflow`).
Scipy is not necessarily needed.
We don't use Six (I think...). Anyway, I guess also already covered implicitly by TF.
Setuptools, Wheels also not needed.
Typing is twice there now.

Reply:
Sure, I can revert these. I had some test failures when scipy was not installed though, so perhaps it should be included there?

Reply:
Where exactly? But anyway, let's move this out of this pull request. Make a separate issue maybe first, including the exception you get/got.